specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/disk.json

```diff
@@ -1735,7 +1735,7 @@
     "EncryptionSetProperties": {
       "properties": {
         "encryptionType": {
-          "$ref": "#/definitions/EncryptionType"
+          "$ref": "#/definitions/DiskEncryptionSetEncryptionType"
```

Comment:
change existing model will be considered as breaking change. suggest to do it in a new API version

Reply:
Hi we can add a comment on the old code saying, the enum value is not valid for disk encryption set. Is adding an optional property in the request object a breaking change? The response already has it